atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/LockWithTimelockEteTest.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * (c) Copyright 2019 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.atlasdb.ete;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import org.junit.Test;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.palantir.atlasdb.lock.LockResource;
+
+public class LockWithTimelockEteTest {
+    private LockResource lockResource = EteSetup.createClientToSingleNode(LockResource.class);
+
+    @Test
+    public void smallTimelockLockSucceeds() throws InterruptedException, JsonProcessingException {
```

Comment:
seems pretty odd to have a `InterruptedException` and a `JsonProcessingException`?

Reply:
Fixed